src/main/xar-resources/modules/indexes.xqm

```diff
@@ -382,7 +631,42 @@ declare function indexes:analyze-lucene-indexes($xconf) {
                             ))}">Node</a>
                             )
                     }</td>
-                </tr>
+                </tr>,
+                for $f in $facets-fields
+                return
+                    <tr>
```

Comment:
I find it easier to read when the rendering of a facet or field would be in a separate function.
either called in a FLWOR-expression like
`return indexes:render-facets-or-fields($f)`
or in `for-each($facets-fields, indexes:render-facets-or-fields#1)`